.github/workflows/lighthouse.yml

```diff
@@ -1,9 +1,10 @@
 name: Audit pull request
-on: pull_request
-
+on:
+  status
 jobs:
   audit:
     runs-on: ubuntu-latest
+    if: github.event.context == 'netlify/deploy-preview' && github.event.state == 'success'
```

Comment:
OH! Wow. I would have never figured that out on my own.

I've used the GitHub API briefly but never to this extent. Let's give this a shot!